docs/install/build.rst

```diff
@@ -176,6 +176,11 @@ Set the repository branch using the variable: ``ROCM_BRANCH``. For example, for
             make -j$(nproc)
             sudo make install
 
+.. note::
+
+   HIP runtime is not buildable on Windows as it depends on closed source
```

Comment:
For this sentence "HIP runtime is not buildable on Windows as it depends on closed source components."
It's better like this,
The instructions are applicable on Linux only.

Reply:
@jujiang-del, you are right. I should avoid negative statements, but I have to highlight that this is currently not buildable on Windows, so I went with the "The HIP runtime is only buildable on Linux."

Yes, the instructions are Linux-specific, but we have to highlight that users should not try to build on Windows.

Reply:
Looks good. Thanks.